src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String SEPARATOR = System.lineSeparator();
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        File fileToRead = new File(fromFileName);
+        File fileToWrite = new File(toFileName);
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileToRead));
+                 BufferedWriter writer = new BufferedWriter(new FileWriter(fileToWrite))) {
```

Comment:
devide logic of reading and writing into a separate methods
Also you don't need File object to create FileReader
```suggestion
              try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
```